src/modules/idle_inhibitor.cpp

```diff
@@ -10,8 +10,13 @@ waybar::modules::IdleInhibitor::IdleInhibitor(const std::string& id, const Bar&
                                               const Json::Value& config)
     : ALabel(config, "idle_inhibitor", id, "{status}"),
       bar_(bar),
+      idle_inhibit_manager_(waybar::Client::inst()->idle_inhibit_manager),
       idle_inhibitor_(nullptr),
       pid_(-1) {
+  if (idle_inhibit_manager_ == nullptr) {
```

Comment:
I prefer to check directly `waybar::Client::inst()->idle_inhibit_manager` rather than creating another variable

Reply:
Fixed.